google/cloud/aiplatform/metadata/artifact.py

```diff
@@ -176,6 +176,7 @@ def _create(
 
         """
         api_client = cls._instantiate_client(location=location, credentials=credentials)
```

Comment:
Maybe it is just a display issue, I saw a duplicated line here.

Reply:
Done.